content/topics/code-reviews/part-1-reviewers/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+content_type: topic
+ready: true
+flavours:
+- any_language
+prerequisites:
+  hard:
+  - why-and-how-to-review-code
+tags:
+- soft skills
+- communication
+title: Reviewer Communication During Code Reviews
+---
+
+The people who's code you review, are people. Remember this. Treat them with respect.
```

Comment:
Three is an extra ,

`The people who's code you review, are people` should be `The people who's code you review are people`